ginga/rv/plugins/LoaderConfig.py

```diff
@@ -0,0 +1,173 @@
+# This is open-source software licensed under a BSD license.
+# Please see the file LICENSE.txt for details.
+"""
+The ``LoaderConfig`` plugin allows you to configure the file openers that
+can be used to load various content into Ginga.
+
+Registered file openers are associated with file MIME types, and there can
+be several openers for a single MIME type.  A priority associated
+with a MIME type/opener pairing determines which opener will be used
+for each type--the lowest priority value will determine which opener will
+be used.  If there are more than one opener with the same low priority
+then the user will be prompted for which opener to use, when opening a
+file in Ginga.  This plugin can be used to set the opener preferences
+and save it to the user's $HOME/.ginga configuration area.
```

Comment:
Would be nice to give an example exactly what the filename should be in `$HOME/.ginga` and some example content of that file.